Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM golang:1.13-alpine as builder
 MAINTAINER FullStory Engineering
 
-# create non-priveleged group and user
-RUN addgroup -S hauser && adduser -S hauser -G hauser
+# create non-priveleged group and user and an owned directory
```

Comment:
```suggestion
# create non-privileged group and user and an owned directory
```

Reply:
Maybe worth a comment that we do this in the builder, instead of below in the target container, because "scratch" doesn't include these utilities.